.env.example

```diff
@@ -0,0 +1,8 @@
+SECRET_KEY_BASE=
```

Comment:
will this be automatically set during the docker setup? or is there a command to run manually?

Reply:
This will now be automatically set in anyways. We will need to generate it. 
We need to run `mix phx.gen.secret` to generate this key. Updating it to Readme as well.